gxawsserverless/src/main/java/com/genexus/cloud/serverless/aws/LambdaHandler.java

```diff
@@ -74,6 +75,12 @@ private void handleSpecialMethods(AwsProxyRequest awsProxyRequest) {
 			qString.add("", parmValue);
 			awsProxyRequest.setMultiValueQueryStringParameters(qString);
 		}
+
+		// In Jersey lambda context, the Referer Header has a special meaning. So we copy it to another Header.
+		List<String> referer = awsProxyRequest.getMultiValueHeaders().get("Referer");
+		if (referer != null && !referer.isEmpty()) {
+			awsProxyRequest.getMultiValueHeaders().put("GX-Referer", referer);
+		}
```

Comment:
And who is going to read the "GX-Referer" ?

Reply:
The Referer Header is required in Angular Applications to get the Query String of the original URL. 

However, due to AWS Lambda Jersey implementation, this Header is override by Lambda specific context.
We need to do a fork/clone or some override in order to get the Referer Header working. In the meantime we are copying it.